ndb/src/google/cloud/ndb/_ports.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Ported implementations from the Google App Engine SDK.
+
+The following members have been brought in:
+
+- ``google.appengine.api.datastore_types.GeoPt``
+"""
+
+
+import functools
+
+from google.cloud.ndb import exceptions
+
+
+@functools.total_ordering
+class GeoPt:
+    """A geographical point, specified by latitude and longitude.
+
+    Typically, latitude and longitude are floating-point coordinates. Often
+    used to integrate with mapping sites like Google Maps. May also be used as
+    ICBM coordinates.
```

Comment:
Intercontinental ballistic missile?!?!

Reply:
Haha I thought the same thing. It's literally that acronym: https://en.wikipedia.org/wiki/ICBM_address

Reply:
:man_shrugging: